### PR TITLE
Exercise 1

### PR DESCRIPTION
**Pull request by Noah-Vincenz**
module 1 exercise

### CODE REVIEW

**Comment by Noah-Vincenz** [comment]
Merged pull request

File: app.py [hunk 1]
```diff
@@ -6,7 +6,20 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=session.get_items())
+
+@app.route('/add', methods=['POST'])
+def add():
+    session.add_item(request.form.get('input_field1'))
```
Comment (Jonesey13) [inline app.py:13]: The name for the input fields here could be more descriptive

File: templates/index.html [hunk 2]
```diff
@@ -10,9 +10,26 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <div class="list-group">
+        {% for item in items %}
+        <a href="#" class="list-group-item list-group-item-action">
```
Comment (Jonesey13) [inline templates/index.html:15]: I'd indent these one level to make it clearer what belongs to the for loop

File: templates/index.html [hunk 3]
```diff
@@ -10,9 +10,26 @@ <h1 class="display-4">To-Do App</h1>
...
+          <div class="d-flex w-100 justify-content-between">
+            <h5 class="mb-1">{{ item.id }}:   {{ item.title }}</h5>
+            <small>{{ item.status }}</small>
+          </div>
+        </a>
+        {% endfor %}
+      </div><br>
+      <form action="/add" method="post">
+        <label for="exampleInput1">Item to be added:</label><br>
+        <input type="text" name="input_field1" placeholder="title" id="exampleInput1">
```
Comment (Jonesey13) [inline templates/index.html:25]: I'd add a `required` attribute here to prevent adding empty todos

File: templates/index.html [hunk 4]
```diff
@@ -10,9 +10,26 @@ <h1 class="display-4">To-Do App</h1>
...
+        <button class="btn btn-outline-primary" type="submit">Add your item</button>
+      </form>
+      <form action="/mark" method="post">
+        <label for="exampleInput2">Item to be marked as 'Completed':</label><br>
```
Comment (Jonesey13) [inline templates/index.html:29]: Might be cleaner to add buttons next to the todo items

**Review by Jonesey13 — COMMENTED** [review]
Looks good to me. Left a few comments but all looks to be working 👍

**Pull request merged**